AppMetrics/UpdatesDialog.qml

```diff
@@ -171,9 +171,9 @@ Dialog {
                             enabled: true
                             background: Rectangle {
                                 anchors.fill: parent
-                                color: Singletons.Config.buttonStates(control)
+                                color: Singletons.Config.buttonStates(parent)
                                 radius: app.info.properties.mainButtonRadius
-                                border.width: control.enabled ? app.info.properties.mainButtonBorderWidth : 0
+                                border.width: parent.enabled ? app.info.properties.mainButtonBorderWidth : 0
```

Comment:
Please refer to the parent element with its `id` instead of `parent`. This increases readability.

Furthermore, change the line https://github.com/Esri/TilePackageKreator/blob/3cabeb26c15f2925904509ed9bb780c579ed2fc0/AppMetrics/UpdatesDialog.qml#L81
to
```
        width: ListView.view.width
```
Using `parent` to access `ListView` properties from inside a delegate is illegal in Qt5.15 and above.